### ISSUE TITLE
Something wrong when i use the “soundstream” repo

### ISSUE THREAD

**Issue by wangyuxuan11** [lucidrains/audiolm-pytorch#184]
Thank you for your excellent code~ When I train the SoundStream on my own data from scratch, I encountered a problem that the training loss became "nan" at around 4k steps, I reduced the initial learning rate by a factor of 10, the "soundstream total loss" degraded from 40 to 13 at 17k steps, but the generated ".flac" file stll contains obvious noise and the voice is not clear at all, just like the picture below! I have no idea but I keep the same parameters as you. Do I need to train for more steps? What is the approximate number of steps to get good audio？Do I need to adjust any parameters？Thanks！
<img width="1312" alt="image" src="https://github.com/lucidrains/audiolm-pytorch/assets/111549316/63595d15-ad3e-484a-8e6c-bc38b5ed107a">

**Comment by Makiyuyuko** [comment]
Hi I'm also training Soundstream and I think we might have met with similar problems. Wondering if you have resolved it? What's your batch_size and audio_sample_rate? Thank you.
